src/test/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotTest.java

```diff
@@ -375,7 +375,7 @@ public void snapshotColocatedNonColocatedThenStream(boolean initialOnly) throws
     }
 
     @ParameterizedTest
-    @ValueSource(booleans = {true, false})
+    @ValueSource(booleans = {false})
```

Comment:
Does this test break colocation tests?

Reply:
This was a test only change, I was running couple of them to verify. I have reverted this too.